kotlin-mbedtls/src/main/kotlin/org/opencoap/ssl/transport/DtlsServer.kt

```diff
@@ -132,9 +132,18 @@ class DtlsServer(
 
     fun numberOfSessions(): Int = executor.supply { sessions.size }.join()
 
-    fun setSessionAuthenticationContext(adr: InetSocketAddress, authenticationContext: String): CompletableFuture<Boolean> =
+    fun setSessionAuthenticationContext(adr: InetSocketAddress, authenticationCredentials: String?): CompletableFuture<Boolean> =
+        updateSessionAuthenticationContext(adr, DtlsSessionContext.DEFAULT_AUTH_CTX_LABEL, authenticationCredentials)
```

Comment:
Lets not use any defaults

Reply:
I agree. that was introduced to have some backward compatibility. But actually probably there is no big need in that.